src/locale/locales/it/messages.po

```diff
@@ -0,0 +1,4017 @@
+
+#: src/view/com/profile/ProfileHeader.tsx:597
+msgid "⚠Invalid Handle"
+msgstr "⚠Nome utente non valido"
+
+#: src/view/com/util/moderation/LabelInfo.tsx:45
+msgid "A content warning has been applied to this {0}."
+msgstr "A questo post è stato applicato un avviso di contenuto {0}."
+
+#: src/lib/hooks/useOTAUpdate.ts:16
+msgid "A new version of the app is available. Please update to continue using the app."
+msgstr "È disponibile una nuova versione dell'app. Aggiorna per continuare a utilizzarla."
+
+#: src/view/com/util/ViewHeader.tsx:83 src/view/screens/Search/Search.tsx:545
+msgid "Access navigation links and settings"
+msgstr "Accedi alle impostazioni di navigazione"
+
+#: src/view/com/pager/FeedsTabBarMobile.tsx:83
+msgid "Access profile and other navigation links"
+msgstr "Accedi al profilo e altre impostazioni di navigazione"
+
+#: src/view/com/modals/EditImage.tsx:299 src/view/screens/Settings.tsx:445
+msgid "Accessibility"
+msgstr "Accessibilità"
+
+#: src/view/com/auth/login/LoginForm.tsx:163 src/view/screens/Settings.tsx:308
+msgid "Account"
+msgstr "Account"
+
+#: src/view/com/profile/ProfileHeader.tsx:293
+msgid "Account blocked"
+msgstr "Account bloccato"
+
+#: src/view/com/profile/ProfileHeader.tsx:260
+msgid "Account muted"
+msgstr "Account  silenziato"
+
+#: src/view/com/modals/ModerationDetails.tsx:86
+msgid "Account Muted"
+msgstr "Account Silenziato"
+
+#: src/view/com/modals/ModerationDetails.tsx:72
+msgid "Account Muted by List"
+msgstr "Account silenziato dalla Lista"
+
+#: src/view/com/util/AccountDropdownBtn.tsx:41
+msgid "Account options"
+msgstr "Opzioni dell'account"
+
+#: src/view/com/util/AccountDropdownBtn.tsx:25
+msgid "Account removed from quick access"
+msgstr "Account rimosso dall'accesso immediato"
+
+#: src/view/com/profile/ProfileHeader.tsx:315
+msgid "Account unblocked"
+msgstr "Account sbloccato"
+
+#: src/view/com/profile/ProfileHeader.tsx:273
+msgid "Account unmuted"
+msgstr "Account non silenziato"
+
+#: src/view/com/auth/onboarding/RecommendedFeedsItem.tsx:150
+#: src/view/com/modals/ListAddRemoveUsers.tsx:264
+#: src/view/com/modals/UserAddRemoveLists.tsx:203
+#: src/view/screens/ProfileList.tsx:791
+msgid "Add"
+msgstr "Aggiungi"
+
+#: src/view/com/modals/SelfLabel.tsx:56
+msgid "Add a content warning"
+msgstr "Aggiungi un avviso sul contenuto"
+
+#: src/view/screens/ProfileList.tsx:781
+msgid "Add a user to this list"
+msgstr "Aggiungi un utente a questo elenco"
+
+#: src/view/screens/Settings.tsx:383 src/view/screens/Settings.tsx:392
+msgid "Add account"
+msgstr "Aggiungi account"
+
+#: src/view/com/composer/photos/Gallery.tsx:119
+#: src/view/com/composer/photos/Gallery.tsx:180
+#: src/view/com/modals/AltImage.tsx:93
+msgid "Add alt text"
+msgstr "Aggiungi testo alternativo"
+
+#: src/view/screens/AppPasswords.tsx:102 src/view/screens/AppPasswords.tsx:143
+#: src/view/screens/AppPasswords.tsx:156
+msgid "Add App Password"
+msgstr "Aggiungi l' App Password"
```

Comment:
Not sure if that is related to something specific but if it is, please ignore this comment

```suggestion
msgstr "Aggiungi la Password per l'App"
```